keyboards/tweetydabird/lotus58/nanoboot/keymaps/default/info.json

```diff
@@ -0,0 +1,13 @@
+{                                                                                                 
```

Comment:
Please remove the unnecessary whitespace from this file using `qmk format-json <file>`.